setup.py

```diff
@@ -56,10 +50,6 @@ def is_platform_mac():
                                               'pytz >= 2011k',
```

Comment:
this can be a bit simpler, e.g. use a common setuptool_kwargs, leaving out datetutil, then add this separately for PY2 and 3